apps/site/pages/docs/api-extensions.mdx

```diff
@@ -114,24 +123,15 @@ const resolvers = {
 export default resolvers
 ```
 
-That's it, you only need to define these files within the vtex folder for the sake of exposing the new fields. The entire GraphQL creation process must be transparent and done in the background by the FastStore.
-
-### How to use
-
-**TBD - After the Consume new fields from pages features be done. Here we can be inspired by these steps:**
-
-1. add the `customData` field in the `Fragment X` from the `Y file`.
-2. run `yarn dev` in the core package.
-3. run `yarn generate` in the core package (with `yarn dev` running).
-4. Get and use the `customData` as you want in Z section.
+That's it, for the sake of exposing the new fields you only need to define these files within the vtex folder. The entire GraphQL creation process is transparent and done in the background by the FastStore CLI.
 
 ---
 
-## Third-party APIs schema extensions
+## Define third-party APIs schema extensions
```

Comment:
```suggestion
## Defining third-party APIs schema extensions
```

Reply:
We used **API Extensions** (with upper case) in the beginning and now **extensions** in lower case. We should define a pattern for it. Maybe just  "extensions".